README.md

```diff
@@ -51,6 +52,9 @@ end
     ```
 
     If you know your shard information at compile time, you can also add that.
+    
+    Note: Be sure to specify your path where you want the shard migrations to go with the `priv` key.  In the example
```

Comment:
Nit: Can we clear up the wording? Maybe something like 

> Configure the path to the shard migration files using the `migrations_path` configuration key.

Reply:
yep